packages/generators/utils/language.ts

```diff
@@ -4,9 +4,10 @@ export enum LangType {
 }
 
 interface IModuleRule extends Object {
```

Comment:
Yeah I think you should rebase from `master`. The interfaces name as default rule can't start with `I`